### PR TITLE
Fix ukrainian language code

### PR DESCRIPTION
**Pull request by m1n0s**
Due to my future lightning talk at 8th of June and according to [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) I've fixed language code for ukrainian version.

Also fix indentations in JSONs according to `.editorconfig`.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -22,10 +22,10 @@ <h1 class="logo">
 
     <ul class="header-links">
       <li class="header-links__item">
-        {% if page.lang != 'ua' %}
-          <a class="header-links__link" href="/">UA</a>
+        {% if page.lang != 'uk' %}
+          <a class="header-links__link" href="/">UK</a>
```
Comment (sudodoki) [inline _includes/header.html:26]: I think this one didn't need to happen

File: _includes/header.html [hunk 2]
```diff
@@ -22,10 +22,10 @@ <h1 class="logo">
...
         {% else %}
-          <span class="header-links__link header-links__link_active">UA</span>
+          <span class="header-links__link header-links__link_active">UK</span>
```
Comment (sudodoki) [inline _includes/header.html:28]: As well as this

**Comment by m1n0s** [comment]
@sudodoki I think it must be like that because of consistency.

Next to ukrainian button we use `EN` - lang code for English and we don't use `US`, `GB`, `AU`, `CA` or whatever country with the English language. 
That's why we must use `UK` for ukrainian button. 

We have standard already and I don't want to create a new one.

**Comment by tyv** [comment]
@m1n0s if I am not wrong, the ICU code for Ukrainian is uk_UA.
http://www.localeplanet.com/icu/uk-UA/index.html
and `UA` is more common and user friendly, so if I am not wrong , I don't see any reasons in that PR and especially in the templates which affecting UI

**Comment by m1n0s** [comment]
@tyv ICU locale compiled as `{{lang_code}}_{{country_code}}` by ISO 639 and ISO 3166.

So if we follow your logic for another ICU locale which is also used for Ukraine `ru_UA` - switcher must be also `UA`, isn't it? I can be confused to see `UA` `UA` `EN` 😄  for example.

More common is not equals more right. This what I will be talking about on my lightning. Standard. We have so much chaos in FE why would we want to create more chaos instead following the standards?

**Comment by tyv** [comment]
from the day 1 when domain `.uk` will belongs to Ukraine I will agree with you.

**Comment by m1n0s** [comment]
Oh c'mon. Domains are about countries, not languages.

**Comment by tyv** [comment]
we will end up with flag images in this discussion at the end

**Comment by m1n0s** [comment]
Or we can just use Alplha-3 codes from the standard which are more understandable for users.
`UKR` `ENG`
But personally, I prefer Alpla-2 code. 
Like other big companies:
https://uk.wikipedia.org
https://www.olx.ua/uk/
https://www.youtube.com/?gl=UA&hl=uk

**Comment by matochu** [comment]
First of all, this is UI issue.
But we can also use such pairs - eng/укр, English/Украінська

**Comment by m1n0s** [comment]
Agreed. I can change UI switcher to `УКР`, `ENG`.
@tyv @sudodoki

**Comment by tyv** [comment]
good!

**Comment by m1n0s** [comment]
#43

**Pull request merged**